server/api/models/recipe.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema(
+  {
+    servings: {
+      type: Number,
+      required: true
+    },
+    prepTime: String,
+    cookTime: String,
+    ingredients: [{
+      measurement: {
+        type: String,
+        enum: ["grams", "tablespoons", "count"], // TODO: Comprehensive enum list
```

Comment:
TODO: We need to fix this before merging

Reply:
Is this plus `"count"` enough? @zackharley @andrewlitt @ColinLMacLeod1 @derekgrossman 

**Volume**
teaspoon (also t or tsp.)
tablespoon (also T, tbl., tbs., or tbsp.)
fluid ounce (also fl oz)
gill (about 1/2 cup)
cup (also c)
pint (also p, pt, or fl pt - Specify Imperial or US)
quart (also q, qt, or fl qt - Specify Imperial or US)
gallon (also g or gal - Specify Imperial or US)
ml, also milliliter, millilitre, cc (and mL only in the US, Canada and Australia).
l, also liter, litre, (and L only in the US, Canada and Australia).
dl, also deciliter, decilitre (and dL only in the US, Canada and Australia).

**Mass and Weight**
pound (also lb or #)
ounce (also oz)
mg (also milligram or milligramme)
g (also gram or gramme)
kg (also kilogram or kilogramme)

**Length**
mm (also millimeter or millimetre)
cm (also centimeter or centimetre)
m (also meter or metre)
inch (also in or ")

Reply:
Also, how should we handle plural vs. singular (E.g., `pound` vs. `pounds`)?

Reply:
Also this is gonna be a **fat** enum list, should I move it out of the schema as a variable?

Reply:
Regarding the plural thing I might just add the plural and singular versions of every unit

Reply:
So a few things:

1. I definitely back making a constant at the top of the file for the enum.
2. I think we should figure out a way to group the same versions of units together. Like even if we have an object on our server that defines plural and singular, I think that that would be better than storing all of the different representations in the enum. 
3. I think that that list you have is a great starting point. We can always add more as we need them

Reply:
Just gonna make a crack at it and we'll see what everyone thinks. I'm also wondering what we should do about Canadian vs. US spelling, i.e., litre vs. liter.